var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -36,6 +36,8 @@ class Llvm(CMakePackage, CudaPackage):
 
     # fmt: off
     version('main', branch='main')
+    version('15.0.2', sha256='dc11d35e60ab61792baa607dff080c993b39de23fb93b3d3369ba15b0601c307')
+    version('15.0.1', sha256='20bccb964e39f604fdc16d1258f94d2053fbdcdab2b2f6d5e20e6095ec403c00')
```

Comment:
Style checks will (or has) failed due to the single quotes that need to be replaced with double.